package.json

```diff
@@ -1,73 +1,74 @@
 {
-  "name": "@guardian/consent-management-platform",
-  "version": "1.1.0-beta.4",
-  "description": "Library of useful utilities for managing consent state across *.theguardian.com",
-  "main": "lib/index.js",
-  "types": "lib/index.d.ts",
-  "scripts": {
-    "build": "yarn clean && tsc && rollup --config",
-    "clean": "rm -rf lib/*",
-    "test": "jest --config jestconfig.json",
-    "lint": "eslint src/**/*.ts src/**/*.tsx",
-    "tsc": "tsc --noEmit",
-    "validate": "yarn tsc && yarn lint && yarn test",
-    "fix": "yarn validate --fix",
-    "prepublishOnly": "yarn validate && yarn build"
-  },
-  "husky": {
-    "hooks": {
-      "pre-push": "yarn validate"
-    }
-  },
-  "repository": "https://github.com/guardian/consent-management-platform.git",
-  "homepage": "https://github.com/guardian/consent-management-platform.git",
-  "author": "George Haberis <george.haberis@guardian.co.uk>",
-  "contributors": [
-    "Ricardo Costa <ricardo.costa@guardian.co.uk>"
-  ],
-  "license": "Apache-2.0",
-  "dependencies": {
-    "@guardian/src-foundations": "^0.2.3",
-    "consent-string": "^1.5.1",
-    "js-cookie": "^2.2.1",
-    "whatwg-fetch": "^3.0.0"
-  },
-  "peerDependencies": {
-    "@emotion/core": "^10.0.21",
-    "react": "^16.10.2"
-  },
-  "devDependencies": {
-    "@babel/core": "^7.6.4",
-    "@babel/plugin-proposal-class-properties": "^7.5.5",
-    "@babel/preset-env": "^7.6.3",
-    "@babel/preset-react": "^7.6.3",
-    "@babel/preset-typescript": "^7.6.0",
-    "@emotion/babel-preset-css-prop": "^10.0.17",
-    "@emotion/core": "^10.0.21",
-    "@types/jest": "^24.0.16",
-    "@types/js-cookie": "^2.2.2",
-    "@types/react": "^16.9.9",
-    "@typescript-eslint/eslint-plugin": "^2.1.0",
-    "@typescript-eslint/parser": "^1.13.0",
-    "eslint": "^6.3.0",
-    "eslint-config-airbnb-typescript": "^4.0.1",
-    "eslint-config-prettier": "^6.0.0",
-    "eslint-plugin-import": "^2.18.2",
-    "eslint-plugin-jest": "^22.14.1",
-    "eslint-plugin-prettier": "^3.1.0",
-    "eslint-plugin-react": "^7.16.0",
-    "husky": "^3.0.2",
-    "jest": "^24.8.0",
-    "prettier": "^1.18.2",
-    "react": "^16.10.2",
-    "rollup": "^1.24.0",
-    "rollup-plugin-babel": "^4.3.3",
-    "rollup-plugin-commonjs": "^10.1.0",
-    "rollup-plugin-node-resolve": "^5.2.0",
-    "ts-jest": "^24.0.2",
-    "typescript": "^3.5.3"
-  },
-  "files": [
-    "lib/**/*"
-  ]
+    "name": "@guardian/consent-management-platform",
+    "version": "2.0.0",
```

Comment:
Saving changed the tab spacing but here are the actual changes to this file:
- Version updated to `2.0.0`.
- Added `rollup-plugin-typescript2` dependency.
- Updated `typescript` dependency to `~3.5.3`